code/game/objects/items/devices/lightreplacer.dm

```diff
@@ -95,8 +97,33 @@
 				user.drop_item()
 				qdel(L)
 				return
-		else
-			to_chat(user, "You need a working light.")
+		else if(L.status == 2 || L.status == 3)
+			AddShards(1)
+			to_chat(user, "You insert [L] into [src]. You have [shards] shards remaining.")
+			user.drop_item()
+			qdel(L)

```

Comment:
Any item you can pick up can be `NODROP`, becuase of the augment that grants it.

Even if we didn't, it's for "Just in case" situations.